src/main/java/quotes/App.java

```diff
@@ -29,28 +38,51 @@ public static void main(String[] args) throws IOException, IllegalStateException
             connection.disconnect();
             System.out.println("Quote From API: " + content);
 
+        }catch (IOException e) {
+            noInternet();
+        }
+    }
 
-            //Appending API quote to JSON file
-            Quote newQuote = new Quote("Ron Swanson", content.toString());
-            Parser parser = new Parser();
-            Quote[] quotesFromFile = parser.jsonReader();
-            FileWriter file = new FileWriter("src/main/resources/recentquotes.json");
-            file.write(content.toString());
-//            file.append(quotesFromFile)
-
+    //Getting Quotes from JAS
+    public static void noInternet() {
```

Comment:
I like that you put all of this in a method called `noInternet()`, it's very clear what is happening